doc/.gitignore

```diff
@@ -1 +1 @@
-xnemogcm*
\ No newline at end of file
+xnemogcm*
```

Comment:
Just curious where this is from?

Reply:
Hmmm... I don't know how this sneaked in (I don't recall doing anything with the `.gitignore` file)

Reply:
This came from when I created the NEMO example ( PR #192 ).